README.md

```diff
@@ -110,6 +110,13 @@ $ nrfjprog --program feather_nrf52832_bootloader.hex -f nrf52
 $ nrfjprog --reset -f nrf52
 ```
 
+#### Burning Particle Xenon Bootloader with [Particle Debugger](https://www.adafruit.com/product/4001)
+*Tested with Windows 10*
+Connect the Particle Debugger to your Particle Xenon, and plug the debugger into your computer. It should show up as a drive named DAPLINK. 
+Copy the bootloader hex file found in your board files in `adafruit\hardware\nrf52\<version>\bootloader\particle_xenon`, onto the DAPLINK drive. It will automatically flash the bootloader, and reset. 
```

Comment:
`found in your board files in adafruit\hardware\nrf52\<version>\bootloader\particle_xenon` is too specific. Other user may use Particle debugger with other board as well, please remove the specific path of xenon. You can use the simply refer to bootloader hex file in 'bin/bootloader' as above nrfjprog does.